### PR TITLE
[2.1] AAP-5802 Add common content to docs (#539)

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Backport #539 to 2.1
* AAP-5802 Add common content to docs

### CODE REVIEW

File: downstream/titles/hub/configuring-private-hub-rh-certified/master.adoc [hunk 1]
```diff
@@ -8,6 +8,8 @@ You can sync {HubName} to use the Red Hat Certified Collections available throug
 
 Your organization can access and curate into a unique set of collections from all Red Hat Certified content in the {HubName} service hosted on console.redhat.com.
 
+include::aap-common/making-open-source-more-inclusive.adoc[leveloffset=+1]
```
Comment (sayjadha) [inline downstream/titles/hub/configuring-private-hub-rh-certified/master.adoc:11]: Suggest having the intro. content under heading 'Overview'.

File: downstream/titles/hub/uploading-collections/master.adoc [hunk 2]
```diff
@@ -8,6 +8,8 @@ Red Hat Automation Hub distributes certified, supported collections from partner
 You can upload collections using the Automation Hub user interface or the `ansible-galaxy` client.
```
Comment (sayjadha) [inline downstream/titles/hub/uploading-collections/master.adoc:8]: Suggest having the intro. content under heading 'Overview'.

File: downstream/titles/hub/getting-started/master.adoc [hunk 3]
```diff
@@ -16,6 +16,8 @@ The Ansible Galaxy client, `ansible-galaxy`, manages roles and collections from
 
 This guide walks you through the steps required to configure your `ansible.cfg` file to use Red Hat {HubName} as the default source for certified Ansible collections content.
```
Comment (sayjadha) [inline downstream/titles/hub/getting-started/master.adoc:17]: Suggest having the intro. content under heading 'Overview'.

File: downstream/titles/hub/high-availability/master.adoc [hunk 4]
```diff
@@ -14,5 +14,7 @@ This configuration eliminates single points of failure to minimize service downt
 This guide covers deployment of a HA {HubName} application stack only.
 Other HA components, such as database and file system HA, or setting up DNS load balancing, are out of scope for this guide.
```
Comment (sayjadha) [inline downstream/titles/hub/high-availability/master.adoc:15]: Suggest having the intro. content under heading 'Overview'.

**Review by sayjadha — COMMENTED** [review]
Hi @ariordan-redhat : I have a few formatting-related suggestions (that might be out of scope of this PR). Rest LGTM. Let me know if you have any questions/disagree with any comment. TY.

**Comment by ariordan-redhat** [comment]
@sayjadha Thanks for the comments. I agree with them, but I think we should gather them into a separate ticket.

**Pull request merged**